site/get-started/validator/quickstart-validator.qmd

```diff
@@ -1,22 +1,179 @@
 ---
-title: "Quickstart — Model Validation [coming soon]{.smallercaps .pink}"
+title: "Quickstart — Model Validation"
+subtitle: "20 minutes"
 date: last-modified
-page-layout: full
+listing:
+  - id: whats-next
+    type: grid
+    grid-columns: 2
+    max-description-length: 250
+    sort: false
+    fields: [title, description]
+    contents:
+    - ../../training/validator-fundamentals/validator-fundamentals-register.qmd
+    - ../../developer/validmind-library.qmd
 ---
 
 Use the {{< var validmind.developer >}} to run tests and validate a model, then attach your test results as evidence to your validation report in the {{< var validmind.platform >}}.
 
-<!-- BECK'S OUTLINE / NOTES -->
-
-<!-- {{< include /get-started/common-assets/_before-you-begin.qmd >}}
+{{< include /get-started/common-steps/_before-you-begin.qmd >}}
 
 ## Register a model for validation
 
+Typically, you will be asked to validate a champion model that has already been independently registered in the model inventory and submitted to you. 
+
+For this quickstart, you'll register a dummy model first and assign yourself as the validator to familiarize you with the {{< var validmind.platform >}} interface:
+
+{{< include /get-started/common-steps/_register-your-first-model.qmd >}}
+
+::: {.column-margin}
+![Registration modal with the options filled out for the model validation quickstart](validator-new-model.png){width=90% fig-alt="A screenshot showing the registration modal with the options filled out for the model validation quickstart" .screenshot}
+:::
+
+#### Assign validator credentials
+
+In order to log tests as a validator instead of as a developer, we'll need to adjust your **model stakeholder** permissions.
+
+On the model details page after you've registered your model:
+
+::: {.panel-tabset}
```

Comment:
In the comments that follow for this file, I suggested to NUMBER the tabs to better indicate that there are procedural steps here.